server/updates.go

```diff
@@ -182,6 +183,16 @@ func (s *Server) checkForUpdates() {
 	}
 	defer res.Body.Close()
 
+	if res.StatusCode != http.StatusOK {
+		b, err := ioutil.ReadAll(res.Body)
+		if err != nil {
```

Comment:
I figured it was useful to mention "Error checking for updates" to give some context to the log, and "Error checking for updates, ..., error: nil" would look kind of silly.